src/main/java/ro/sapientia/furniture/model/Order.java

```diff
@@ -0,0 +1,30 @@
+package ro.sapientia.furniture.model;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+
+import lombok.Data;
+
+@Entity(name="order")
+@Data
+public class Order implements Serializable{
+	
+	private static final long serialVersionUID = 1L;
```

Comment:
Please add the needed annotations to auto-generated id

Reply:
And please also add getters and setters

Reply:
I used lombok so the getters and setters will be generated automatically